malsim/scenario.py

```diff
@@ -201,7 +200,7 @@ def apply_scenario_node_property_rules(
 
 def apply_attacker_entrypoints(
         attack_graph: AttackGraph, attacker_name: str, entry_points: dict
-) -> None:
+) -> int:
     """Apply attacker entrypoints to attackgraph from scenario
 
     Creater attacker, add entrypoints to it and compromise them.
```

Comment:
The method should be renamed and the docstrings should be updated. I also think that it might be a reasonable idea to just return the Attacker that was created instead. But I really like the idea behind the change.

Reply:
I thikn this is a comment for Joakim's PR. I just changed the return type annotation.

Reply:
Right! But what should this be renamed to? What is wrong here?

Reply:
`Apply` to me suggestions a non-return methods. Now, don't get me wrong, `add` also does that and we have return values on plenty of `add` methods, but I prefer it.

Reply:
I tried renaming this and have it return an Attacker and then got into the rabbit hole of using `attacker_id` instead of `Attacker` objects and trying to fix that and the code became so convoluted that it just did not make sense to me. I will leave this to you or Joakim to fix as you please. My suspicion is that this will be greatly simplified when the Attacker class is removed (or whatever it ends up being with node.compromise/attacker.compromise).